src/core/sequence_batch_scheduler.cc

```diff
@@ -993,6 +1000,8 @@ DirectSequenceBatch::SchedulerThread(
         //      b) the required tensor shapes for the batch for the
         //      case where ragged batching is not allowed
```

Comment:
Add "3) Determine the earliest enqueue time and number of ready sequences if queue delay is enabled"